Sources/Group.php

```diff
@@ -2471,32 +2477,32 @@ protected static function canDelete(object $group): bool
 	/**
 	 * Returns whether the type of the given group can be changed.
 	 *
-	 * @param object $group An instance of this class.
+	 * @param self $group An instance of this class.
 	 * @return bool Whether the group can be deleted.
 	 */
-	protected static function canChangeType(object $group): bool
+	protected static function canChangeType(self $group): bool
 	{
 		return $group->id > self::NEWBIE;
 	}
 
 	/**
 	 * Returns whether the permissions of the given group can be changed.
 	 *
-	 * @param object $group An instance of this class.
+	 * @param self $group An instance of this class.
 	 * @return bool Whether the group can be deleted.
 	 */
-	protected static function canChangePermissions(object $group): bool
+	protected static function canChangePermissions(self $group): bool
 	{
 		return $group->id !== self::ADMIN;
 	}
 
 	/**
 	 * Returns whether people can search for members of this group.
 	 *
-	 * @param object $group An instance of this class.
+	 * @param self $group An instance of this class.
 	 * @return bool Whether the group can be deleted.
 	 */
-	protected static function canSeach(object $group): bool
+	protected static function canSeach(self $group): bool
```

Comment:
Possible type in the method name? I would expect canSearch but can recall what it used to be.

Reply:
Its compat code:
```
		'can_search' => __CLASS__ . '::canSearch',
```
Just a typo.  I corrected it